cwl_utils/parser/__init__.py

```diff
@@ -8,9 +8,12 @@
 from schema_salad.exceptions import ValidationException
 from schema_salad.utils import yaml_no_ts
 
-from . import cwl_v1_0 as cwl_v1_0
-from . import cwl_v1_1 as cwl_v1_1
-from . import cwl_v1_2 as cwl_v1_2
+from cwl_utils.errors import GraphTargetMissingException
+from cwl_utils.parser import (
+    cwl_v1_0 as cwl_v1_0,
+    cwl_v1_1 as cwl_v1_1,
+    cwl_v1_2 as cwl_v1_2,
+)
```

Comment:
@GlassOfWhiskey , here's how I fixed the `make html` (and the Sphinx command used in RTD) in my local environment:

```diff
diff --git a/cwl_utils/parser/__init__.py b/cwl_utils/parser/__init__.py
index 5a28cdc..19164b9 100644
--- a/cwl_utils/parser/__init__.py
+++ b/cwl_utils/parser/__init__.py
@@ -8,12 +8,11 @@ from urllib.parse import unquote_plus, urlparse
 from schema_salad.exceptions import ValidationException
 from schema_salad.utils import yaml_no_ts
 
-from cwl_utils.errors import GraphTargetMissingException
-from cwl_utils.parser import (
-    cwl_v1_0 as cwl_v1_0,
-    cwl_v1_1 as cwl_v1_1,
-    cwl_v1_2 as cwl_v1_2,
-)
+from ..errors import GraphTargetMissingException
+
+from . import cwl_v1_0 as cwl_v1_0
+from . import cwl_v1_1 as cwl_v1_1
+from . import cwl_v1_2 as cwl_v1_2
 
 LoadingOptions = Union[
     cwl_v1_0.LoadingOptions, cwl_v1_1.LoadingOptions, cwl_v1_2.LoadingOptions

```

Reply:
That worked, thank you @kinow!